R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -971,10 +971,15 @@ test_that("spark.randomForest Classification", {
   predictions <- collect(predict(model, data))$prediction
   expect_equal(length(grep("1.0", predictions)), 50)
   expect_equal(length(grep("2.0", predictions)), 50)
+
+  # spark.randomForest classification can work on libsvm data
+  data <- read.df(absoluteSparkPath("data/mllib/sample_multiclass_classification_data.txt"),
+                source = "libsvm")
+  model <- spark.randomForest(data, label ~ features, "classification")
+  expect_equal(summary(model)$numFeatures, 4)
```

Comment:
BTW, I only add tests of libsvm data for ```spark.gbt``` and ```spark.randomForest```. This PR fix the exception for ```spark.naiveBayes``` and ```spark.glm``` also, I did not add tests for those algorithms because we fix the issue at ```RWrapperUtils.checkDataColumns``` which was called by all algorithms. It means that if one algorithm works well, others should also work well.